ansible-ipi-install/roles/installer/templates/install-config.j2

```diff
@@ -32,7 +32,7 @@ platform:
     ingressVIP: {{ ingressvip }}
     dnsVIP: {{ dnsvip }}
 {% if (release_version[0]|int > 4) or ((release_version[0]|int == 4) and (release_version[2]|int > 3)) %}
-    provisioningNetworkInterface: {{ prov_nic }}
+    provisioningNetworkInterface: {{ masters_prov_nic }}
```

Comment:
Confirming - this only applies to masters?

Reply:
According to the documentation[1]:

```
The name of the network interface on control plane nodes connected to the provisioning network.
```
AFAIK it is 'only' used for the metal3 pod to bind to the proper interface to listen to dhcp requests, serve images, etc.

[1] https://github.com/openshift/installer/blob/master/docs/user/metal/install_ipi.md#required-inputs